TheWarWithin/HunterBeastMastery.lua

```diff
@@ -1294,26 +1391,39 @@ for i = 1, 40 do
 end
 
 spec:RegisterHook( "COMBAT_LOG_EVENT_UNFILTERED", function( _, subtype, _, sourceGUID, sourceName, _, _, destGUID, destName, destFlags, _, spellID, spellName )
+    if sourceGUID == GUID then
+        -- Keep the cycle synced to real game data if we lose it somehow
+        if subtype == "SPELL_AURA_APPLIED" or subtype == "SPELL_AURA_REFRESH" then
+            -- Check if one of the summon buffs is applied and sync the index.
+            for index, buffName in ipairs( pack_leader__buff_cycle ) do
+                if spellID == spec.auras[ buffName ].id then
+                    -- Align the cycle to the buff AFTER the currently applied one.
+                    pack_leader_buff_current = ( index % #pack_leader__buff_cycle ) + 1
```

Comment:
Update the camel-case one here.  Make sure the table declaration for `pack_leader__buff_cycle` is located before the combat log handler in this file.